node/typings.d.ts

```diff
@@ -100,6 +100,7 @@ interface Organization {
   paymentTerms: PaymentTerm[]
   status: string
   created: string
+  customFields: CustomField[] | null
```

Comment:
Why are we adding null to the type here but removing it from the schema? I'd think those should match up.

Reply:
✅I couldn't reproduce the issue that I mention on the standup, so removed the null